build.md

```diff
@@ -2,19 +2,20 @@
 
 Every CFPB project with a non-trivial front-end should:
 
- 1. Use **Grunt** to accomplish all front-end build tasks.
+ 1. Use **Grunt** to accomplish all front-end build tasks - [example Gruntfile](https://github.com/cfpb/generator-cf/blob/master/app/templates/_Gruntfile.js).
  1. Use **Less** as a CSS pre-processor.
  1. Use an optional **config.json** file in the same directory as the gruntfile to store build variables (internal API endpoints, PII, etc.). This file should be consumed by a Grunt task and listed in `.gitignore`.
- 1. Only require **two commands** to install dependencies and build all static assets:
+ 1. Only require **three commands** to install dependencies and build all static assets:
    1. `npm install`
+   1. `bower install`
```

Comment:
It used to be that `bower install` messed up Capital Framework installed dependencies. Is that no longer the case?